OpenTabletDriver.Console/Program.Commands.cs

```diff
@@ -19,6 +20,35 @@ namespace OpenTabletDriver.Console
 {
     partial class Program
     {
+
+        #region STDIO
+            
+        static async Task stdioCommands()
+        {   
+            String stdiocmd;
+
+            var root = new RootCommand("OpenTabletDriver Console Client")
+            {
+                Name = "otd"
+            };
+            root.AddRange(GenerateIOCommands());
+            root.AddRange(GenerateActionCommands());
+            root.AddRange(GenerateDebugCommands());
+            root.AddRange(GenerateModifyCommands());
+            root.AddRange(GenerateRequestCommands());
+            root.AddRange(GenerateListCommands());
+            root.AddRange(GenerateScriptingCommands());
+
+
+            while(true) {
+                stdiocmd = In.ReadLine();
```

Comment:
Remove the early definition and perform readline asynchronously.
```suggestion
                string stdiocmd = await In.ReadLineAsync();
```